test/compatibility/meta/versions.sh

```diff
@@ -6,12 +6,12 @@
 
 # Currently this is a temporary setup for python testsuite
 # This will be merged with the following variables in the future
-export SUITE_RELEASE_BRANCHES="develop mongodb-7.1 mongodb-7.0 mongodb-6.3"
```

Comment:
Question -- why do we no longer need 7.1 or 6.3?

Reply:
As they are both rapid releases that at the time a newer rapid release (e.g. mongodb-7.2) comes out they become deprecated. The short-term support nature for rapid releases is the reason we generally only covers LTS releases in compatibility tests.